lib.typ

```diff
@@ -286,7 +286,7 @@
           #set text(
             size: 32pt,
             style: "normal",
-            font: ("Roboto"),
+            font: font,
```

Comment:
Could we instead have a new parameter for the header font? Personally I like the look of the different font being used for the header. You could name it `header-font`

Reply:
Good idea